unit-tests-jest/numbers.test.js

```diff
@@ -0,0 +1,78 @@
+// import {evenNumbers, toDollars, divideBy, multiplyBy} from './numbers';
+
+const testingEvenNumbers = require('./numbers');
+
+describe('Even Numbers tests', () => {
+  test('[2,3,4,5,6] should return [2,4,6]', () => {
+    const result = testingEvenNumbers.evenNumbers([2, 3, 4, 5, 6]);
+    expect(result).toStrictEqual([2, 4, 6]);
```

Comment:
It's more common to just use `toEqual`. We usually use `toStrictEqual` only in rare cases where we need `===` equality.